string request
geometry_msgs/Point point
uint32 num_pts
float32 tolerance
float32 distance
---
geometry_msgs/Pose[] pose_targets
JointTarget[] joint_targets
bool condition
